src/main/java/com/ciscospark/Client.java

```diff
@@ -206,34 +206,54 @@ private void log(Level level, String msg, Object... args) {
     private <T> Response doRequest(URL url, String method, T body) {
         try {
             HttpsURLConnection connection = getConnection(url);
-            String trackingId = connection.getRequestProperty(TRACKING_ID);
+
+            String requestTrackingId = connection.getRequestProperty(TRACKING_ID);
+            System.out.println("Response Tracking Id: " + requestTrackingId);
```

Comment:
```suggestion
            System.out.println("Request Tracking Id: " + requestTrackingId);
```

Reply:
Yeah, Ill have to figure out how the java logger works. 

```
private void log(Level level, String msg, Object... args) {
        if (logger != null && logger.isLoggable(level)) {
            logger.log(level, msg, args);
        }
    } 
```
    
Do you happen to know what the `level` in the logger.log refers to?